pkg/run/watch/forward_port.go

```diff
@@ -126,3 +137,50 @@ func ForwardPort(log logr.Logger, pod *corev1.Pod, cfg *rest.Config, specMap *Po
 
 	return fw.ForwardPorts()
 }
+
+func ShowPortForwards(log clilogger.Logger, ctx context.Context, portForwards map[rune]PortForwardShortcut) {
```

Comment:
```suggestion
func ShowPortForwards(ctx context.Context, log clilogger.Logger, portForwards map[rune]PortForwardShortcut) {
```
Having the context as first param is more idiomatic.